evm_trace/_display.py

```diff
@@ -0,0 +1,66 @@
+from typing import TYPE_CHECKING, Iterable, Optional
+
+if TYPE_CHECKING:
+    from evm_trace.base import CallTreeNode
+
+
+class DisplayableCallTreeNode(object):
+    _FILE_MIDDLE_PREFIX = "├──"
+    _FILE_LAST_PREFIX = "└──"
+    _PARENT_PREFIX_MIDDLE = "    "
+    _PARENT_PREFIX_LAST = "│   "
```

Comment:
Completely eschewing `rich.tree`?

Reply:
Only in this package! We will use it in ape, as it has lots of nice features such as  markup terminal tags. But I didn't want to take on the dependency here.